src/pages/hcp/api-docs/vault-secrets/[[...page]].tsx

```diff
@@ -3,150 +3,78 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
+// Lib
+import { fetchCloudApiVersionData } from 'lib/api-docs/fetch-cloud-api-version-data'
 // View
-import ApiDocsView from 'views/api-docs-view'
-import {
-	getApiDocsStaticProps,
-	getApiDocsStaticPaths,
-	ApiDocsParams,
-} from 'views/api-docs-view/server'
-import { buildApiDocsBreadcrumbs } from 'views/api-docs-view/server/get-api-docs-static-props/utils'
-import { fetchCloudApiVersionData } from 'views/api-docs-view/utils'
-import { isDeployPreview } from 'lib/env-checks'
-// Revised view
 import OpenApiDocsView from 'views/open-api-docs-view'
 import {
-	getStaticPaths as getOpenApiDocsStaticPaths,
+	getStaticPaths,
 	getStaticProps as getOpenApiDocsStaticProps,
 } from 'views/open-api-docs-view/server'
-// Components
-import {
-	PathTruncationAside,
-	truncateVaultSecretsOperationPath,
-} from 'views/api-docs-view/components'
 // Types
-import type { OperationObjectType } from 'components/open-api-page/types'
-import type { ApiDocsViewProps } from 'views/api-docs-view/types'
-import type {
-	GetStaticPaths,
-	GetStaticProps,
-	GetStaticPropsContext,
-} from 'next'
+import type { GetStaticProps, GetStaticPropsContext } from 'next'
 import type { OpenAPIV3 } from 'openapi-types'
 import type {
+	OpenApiDocsParams,
 	OpenApiDocsViewProps,
-	OpenApiNavItem,
+	OpenApiDocsPageConfig,
 } from 'views/open-api-docs-view/types'
 
 /**
- * 🚩 Flag to use the work-in-progress revised API docs view & server functions.
- */
-const USE_REVISED_TEMPLATE =
-	__config.flags.enable_hcp_vault_secrets_api_docs_revision
-
-/**
- * The product slug is used to fetch product data for the layout.
- */
-const PRODUCT_SLUG = 'hcp'
-
-/**
- * The baseUrl is used to generate
- * breadcrumb links, sidebar nav levels, and version switcher links.
+ * OpenApiDocsView server-side page configuration
  */
-const BASE_URL = '/hcp/api-docs/vault-secrets'
-
-/**
- * Resource items are shown in the sidebar
- */
-const NAV_RESOURCE_ITEMS: OpenApiNavItem[] = [
-	{
-		title: 'Tutorial Library',
-		href: '/tutorials/library?product=vault&edition=hcp',
-	},
-	{
-		title: 'Certifications',
-		href: '/certifications/security-automation',
-	},
-	{
-		title: 'Community',
-		href: 'https://discuss.hashicorp.com/',
+const PAGE_CONFIG: OpenApiDocsPageConfig = {
```

Comment:
I've refactored this page file to remove the conditional previously controlled by the `enable_hcp_vault_secrets_api_docs_revision` flag.

I've also collected all page-specific configuration in this `PAGE_CONFIG` variable. Intent is to make it easier to copy-pasta this page file to a new location, and be able to modify all the config needed to stand up a new product's OpenApiDocsView in one place.